start.py

```diff
@@ -293,6 +293,19 @@ def set_jvm_memory(m2ee_section, vcap, java_version):
         )
 
 
+def set_jetty_config(m2ee):
+    jetty_config_json = os.environ.get('JETTY_CONFIG')
+    if not jetty_config_json:
+        return None
+    try:
+        jetty_config = json.loads(jetty_config_json)
+        jetty = m2ee.config._conf['m2ee']['jetty']
+        jetty.update(jetty_config)
+        logger.debug('Jetty configured: %s ' % json.dumps(jetty))
```

Comment:
Use the comma-based syntax for logging because otherwise it creates the string even when not required:
https://stackoverflow.com/a/34634301/2230226

tl;dr:
`logger.debug('foo is %s', foo)` vs `logger.debug('foo is %s' % foo)`
In #1 the string is lazily constructed, #2 is not.